### DISCUSSION TITLE
What does compute ring mean

### DISCUSSION THREAD

**Discussion by lukar-debug** [Inori/GPCS4#145] [Q&A]
I want to know what it means as I want to practice 3d simulations and graphics

**Comment by VelocityRa** [comment]
See 31:55 here: https://www.youtube.com/watch?v=QMiubC6LdTA

And more info in AMD/Radeon documentation manuals/guides.

**Comment by lukar-debug** [comment]
On Wed, Nov 30, 2022 at 2:47 PM Nick Renieris ***@***.***>
wrote:

> See 31:55 here: https://www.youtube.com/watch?v=QMiubC6LdTA
>
> And more info in AMD/Radeon documentation manuals/guides.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Inori/GPCS4/discussions/145#discussioncomment-4276882>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUFYDYCOSVTHTUJ2AJA7CCDWK6VM7ANCNFSM6AAAAAASOUGEYM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
Thanks
